src/com/itacademy/eshop/product/Description.java

```diff
@@ -0,0 +1,31 @@
+package com.itacademy.eshop.product;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Description {
+
+    private String productDescription;
+    private String descriptionAdditionTime;
```

Comment:
Working with dates. I suggest using Objects which represent date. In your case you can use LocalDateTime. It is good to use specific objects for specific things. Even though you could store a lot of different types of values in a string